packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -490,39 +489,6 @@ const schema: SchemaType<"Users"> = {
     },
     // unique: true // note: find a way to fix duplicate accounts before enabling this
   },
-  // The user's profile URL slug // TODO: change this when displayName changes
-  // Unique user slug for URLs, copied over from Vulcan-Accounts
-  slug: {
-    type: String,
-    optional: true,
-    canRead: ['guests'],
-    canUpdate: ['admins'],
-    order: 40,
-    group: formGroups.adminOptions,
-    
-    onCreate: async ({ document: user }) => {
-      // create a basic slug from display name and then modify it if this slugs already exists;
-      const displayName = createDisplayName(user);
-      const basicSlug = slugify(displayName);
-      return await getUnusedSlugByCollectionName('Users', basicSlug);
-    },
-    onUpdate: async ({data, oldDocument}) => {
-      if (data.slug && data.slug !== oldDocument.slug) {
-        const slugLower = data.slug.toLowerCase();
-        const isUsed = !oldDocument.oldSlugs?.includes(slugLower) && await slugIsUsed("Users", slugLower)
-        if (isUsed) {
-          throw Error(`Specified slug is already used: ${slugLower}`)
```

Comment:
Does this imply we want `onCollision: 'rejectNewDocument'` in `addSlugCallbacks` for `Users`?  (I'm a little confused by what the `if (data.displayName && data.displayName !== oldDocument.displayName)` is doing, so maybe not.)

Reply:
Hmm. It looks like this will throw an error if you explicitly change the slug to a collision, but not if you change the `displayName` to something that autogenerates a slug that is a collision. Which makes sense, but it needs a new option added to `slugUtil` to get that behavior.